src/test/java/com/google/sps/servlets/EventServletTest.java

```diff
@@ -263,4 +280,23 @@ public void eventServletTest_doGet_returnedEventsOutputCorrectly() throws IOExce
     writer.flush();
     assertTrue(stringWriter.toString().contains(new Gson().toJson(EVENT_LIST)));
   }
+
+  @Test
+  public void eventServletTest_getCredentialsFromUserList_returnsValidCredentials() throws InterruptedException,
+      IOException {
+    when(mockFlow.loadCredential("1")).thenReturn(new Credential(BearerToken.authorizationHeaderAccessMethod()).setAccessToken("1"));
+    when(mockFlow.loadCredential("2")).thenReturn(new Credential(BearerToken.authorizationHeaderAccessMethod()).setAccessToken("2"));
+    when(mockFlow.loadCredential("3")).thenReturn(new Credential(BearerToken.authorizationHeaderAccessMethod()).setAccessToken("3"));
```

Comment:
Maybe add a prefix to the strings to make clear what they are for, e.g. "cred1" and "token1"